model/types.go

```diff
@@ -285,6 +286,19 @@ type BlockDeviceAccessInfo struct {
 	AccessProtocol string   `json:"access_protocol,omitempty"`
 	TargetNames    []string `json:"target_names,omitempty"`
 	LunID          int32    `json:"lun_id,omitempty"`
+	SecondaryBackendDetails
+	IscsiAccessInfo
+}
+
+// Information of LUN id, IQN, discovery IP's the secondary array
+type SecondaryBackendDetails struct {
+	PeerArrayDetails []SecondaryLunInfo
```

Comment:
We typically do a []*SecondaryLunInfo.  Can you see above?

Reply:
i will incorporate this